java/api/src/main/java/com/exadel/frs/core/trainservice/component/classifiers/EuclideanDistanceClassifier.java

```diff
@@ -113,7 +113,13 @@ private INDArray normalize(final INDArray embeddings) {
     }
 
     private INDArray calculateSimilarities(INDArray distance) {
-        return tanh(distance.rsubi(1.1).muli(2.5), false).addi(1).divi(2);
+        FacesStatusResponse status = facesApiClient.getStatus();
+        if (status == null || status.getSimilarityCoefficients() == null || status.getSimilarityCoefficients().isEmpty()) {
+            throw new FacesServiceException("No status information received");
```

Comment:
Could you change it to:
Error during synchronization between servers: No status information received

Reply:
Indeed the exception constructor creates error response with the message _Error during synchronization between servers: No status information received_